### DISCUSSION TITLE
livewire-sortable conflict

### DISCUSSION THREAD

**Discussion by Jam0r85** [filamentphp/filament#4095] [Help]
I'm trying to use @nextapps-be/livewire-sortablejs to sort some photos in a livewire component however i've been getting a

'Uncaught Livewire: Directive already registered: [sortable]'

JS error and after some searching I believe it's because the filament form package registers this already.

Any ideas on how to get around this?

**Comment by danharrin** [comment]
You're correct, we do already have an integration of this library in Filament:

https://github.com/filamentphp/filament/blob/2.x/packages/forms/resources/js/sortable.js#L5-L11

However, we can move to the Alpine version instead. We have plans to remove the Livewire version from core in v3.

So if you only need a simple sortable integration, `x-sortable`, `x-sortable-item` and `x-sortable-handle` Alpine directives should already work within Filament.

**Reply by Jam0r85** [reply]
Is there anyway to 'tap' into the current implementation like in the above package? eg. calling a livewire component method after an item has been moved? Is there an event that can be listened to?

**Reply by danharrin** [reply]
With `x-sortable`, you can use `x-on:end` as Sortable.js will emit the `end` event
